src/raftstore/store/peer.rs

```diff
@@ -1723,14 +1773,9 @@ impl Peer {
     }
 
     pub fn get_min_progress(&self) -> u64 {
-        let status = self.raft_group.status();
-        status
-            .progress
-            .values()
-            .chain(status.learner_progress.values())
-            .map(|pr| pr.matched)
-            .min()
-            .unwrap_or_default()
+        self.raft_group.status().progress.map_or(0, |p| {
```

Comment:
Learners should be considered. BTW it's better to comment it to avoid make mistake again.

Reply:
The branch I use returns a ProgressSet instead of separating progresses. So it contains both voters and learners.